wremnants/muon_efficiencies_veto.py

```diff
@@ -36,7 +37,14 @@ def make_muon_efficiency_helpers_veto(filename_plus = data_dir + "/muonSF/smooth
     effSyst_decorrEtaEdges = [round(-2.4 + 0.1*i,1) for i in range(49)]
     Nsyst = 1 + (len(effSyst_decorrEtaEdges) - 1) # 1 inclusive variation + all decorrelated bins
 
-    Steps = 3 #we decided to compute the syst variations on the veto SFs independently for each of the tnp fits (using only global muons in the muon definition we fit reco, "tracking", looseID + dxybs)
+    if useGlobalOrTrackerVeto: #in this way we are hardcoding the file names for global-or-tracker, but I don't think we are going to change them in the helpers anyways
+        filename_plus = data_dir + "/muonSF/smoothedSFandEffi_newveto_globalortracker_regular_GtoH_plus.root"
```

Comment:
Perhaps for symmetry reasons at this point you can also hardcode the files for global inside this function. Mainly because we don't really expect to change them intermittently, and I see you don't pass them to the helper explicitly.